src/AppBundle/Resources/views/Site/edit.html.twig

```diff
@@ -10,6 +10,176 @@
     </div>
 {% endblock headerPanel %}
 
-{% block body -%}
+{% block body %}
     {{ form(edit_form) }}
 {% endblock %}
+
+{% block javascripts %}
+    <script>
```

Comment:
wha???

You'll be wanting to move this into a js file in the web-src directory